resources/js/Pages/Collections/Components/CollectionNavigation/CollectionNavigation.tsx

```diff
@@ -152,11 +152,11 @@ export const CollectionNavigation = ({
                                 content={updateDisabledReason()}
                                 disabled={!isTruthy(updateDisabledReason())}
                             >
-                                <div className="py-1">
+                                <div className="flex items-center sm:pr-8 lg:pr-6">
                                     <Button
                                         icon="Refresh"
                                         variant="secondary"
-                                        className="hidden bg-theme-secondary-200 text-theme-primary-900 sm:block"
+                                        className="hidden h-10 bg-theme-secondary-200 text-theme-primary-900 sm:block"
```

Comment:
why is this button larger than `h-10` to begin with? 🤔 might be worth finding the reason that causes that than forcing it to 40 as the secondary button by default should already be 40px

Reply:
I still see `h-10` here